### ISSUE TITLE
fonts-shadow on selected vs. no-shdow 

### ISSUE THREAD

**Issue by Paz-it** [ubuntu/gtk-communitheme#475]
fonts-shadow : 
![screenshot from 2018-05-23 22-46-55](https://user-images.githubusercontent.com/37431503/40448104-37d3ce00-5edd-11e8-88fa-aa4812f156a2.png)
no-shadow : 
![screenshot from 2018-05-23 22-49-13](https://user-images.githubusercontent.com/37431503/40448147-57c670b4-5edd-11e8-9184-74f799f5231b.png)
IMO, better with shadow on selected but also inconsistency. 
I apologize in advance if I'm repeating a known issue, I find it hard to follow.. :)

**Comment by clobrano** [comment]
Hi @Paz-it, this is probably a new topic :smile: 
One question, is there a place where we use shadow, or is it just a mockup?

**Comment by Paz-it** [comment]
Hi @clobrano ..No mockup here -- The one with shadow is a selected screenshot on the desktop.

**Comment by clobrano** [comment]
For consistency, I think better without shadow

**Comment by madsrh** [comment]
Nicely spotted @Paz-it, but I think I agree with the @clobrano above.

**+1** for leaving things as they are. Some might say that desktop icons are[ a thing of the past](https://www.omgubuntu.co.uk/2018/01/gnome-desktop-icons-removed-3-28) 😜

**Comment by Paz-it** [comment]
Hi  @madsrh !   My example was to show the font-shadow on selected. it looks much better IMHO.
Icons on desktop is from a different opera :)))

**Comment by clobrano** [comment]
@Paz-it I understood the point, I just said that no font-shadow was a decision we made for other widgets as well, since it goes against the flat theme.
